ci_framework/roles/openshift_setup/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create required namespaces
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_kubeconfig }}"
+    name: "{{ item }}"
+    kind: Namespace
+    state: present
+  with_items: >
+    {{
+      [cifmw_install_yamls_defaults['NAMESPACE']] + (
```

Comment:
that will crash in case ppl are using the module without the install_yamls wrapper. We should stick to a "be generic" approach, and set defaults whenever we need the.